superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx

```diff
@@ -89,19 +88,21 @@ class VisualizeModal extends React.PureComponent {
           </div>);
       }
     });
+
+    const getNumColsHaveProperty = propName =>
```

Comment:
would `hasColumnType` be more clear here? i would also suggest moving this to it's own method on the component.